docs/conf.py

```diff
@@ -51,6 +51,8 @@
     "myst_nb",
     "matplotlib.sphinxext.plot_directive",
     "sphinx_tabs.tabs",
+    "pydata_sphinx_theme",
```

Comment:
this is already in the requirements so it is installed and is the value of `html_theme`. It should not be added as an extension. The best case scenario is nothing happens, worst case scenario is it breaks the doc build